oak_functions/proto/abi.proto

```diff
@@ -46,6 +46,10 @@ enum OakStatus {
   ERR_STORAGE_ITEM_NOT_FOUND = 4;
   // Error when running the TensorFlow model, due to bad input tensor.
   ERR_BAD_TENSOR_FLOW_MODEL_INPUT = 5;
+  // Error when extension handle is invalid.
+  ERR_INVALID_EXTENSION_HANDLE = 6;
+  // Error when extension is not available.
+  ERR_EXTENSION_NOT_AVAILABLE = 7;
```

Comment:
I'm not sure I get the difference between an invalid extension handle vs an extension not available, could you clarify?